webclient/src/i18n-backend.ts

```diff
@@ -0,0 +1,16 @@
+import { ModuleType } from 'i18next';
+
+import { Language } from 'types';
+
+class I18nBackend {
```

Comment:
@brentmclark maybe you know a better way to do this than creating our own `backend` plugin.